source/RialtoGStreamerMSEBaseSink.cpp

```diff
@@ -1019,24 +1024,127 @@ void rialto_mse_base_sink_lost_state(RialtoMSEBaseSink *sink)
     gst_element_lost_state(GST_ELEMENT_CAST(sink));
 }
 
-bool rialto_mse_base_sink_get_n_streams_from_parent(GstObject *parentObject, gint &n_video, gint &n_audio)
+static bool rialto_mse_base_sink_get_n_streams_from_parent(GstObject *parentObject, gint &n_video, gint &n_audio,
+                                                           gint &n_text)
 {
     if (g_object_class_find_property(G_OBJECT_GET_CLASS(parentObject), "n-video") &&
-        g_object_class_find_property(G_OBJECT_GET_CLASS(parentObject), "n-audio"))
+        g_object_class_find_property(G_OBJECT_GET_CLASS(parentObject), "n-audio") &&
+        g_object_class_find_property(G_OBJECT_GET_CLASS(parentObject), "n-text"))
```

Comment:
This code is no longer executed in our unittests because we don't set n-text property, set property in RialtoGstTest::installAudioVideoStreamsProperty?

Reply:
This code is not executed, because I disabled UTs executing it in one of my previous deliveries. There's ticket RIALTO-568 to reenable it.